client/src/containers/profile/profile.js

```diff
@@ -4,8 +4,16 @@ export default class Profile extends React.Component {
   render() {
     return (
       <div>
-        <p>Replace me!</p>
+        <h1>{this.props.teamName} </h1>
+        <b>{this.props.primaryContactName} </b>
```

Comment:
According to footnote on [this](https://www.w3schools.com/tags/tag_b.asp) page:
```
...the <b> tag should be used as a LAST resort when no other tag is more appropriate.
```
You can make some text bold by adding the css rule `font-weight: bold;`. Remember that adding style in React is a wee bit different, so check out the [this article](https://www.reactenlightenment.com/react-jsx/5.6.html) on how to do that. (note that this uses `var`, but you should be using `const` if you don't plan on changing that variable (likely the case here) or `let` if you want to change it)

That line should end up looking something like this:
```
<div style={/* insert style to make text bold here */}>{this.props.primaryContactName}</div>
```

Reply:
The irony is in that note, they use the `<b>` tag.
![image](https://user-images.githubusercontent.com/2314816/30551561-72eb5192-9c60-11e7-8fbb-b9d14c12af6a.png)